api/document/models.py

```diff
@@ -46,12 +47,18 @@ class DocNode(models.Model):
     '''
 
     policy = models.ForeignKey('reqs.Policy', on_delete=models.CASCADE)
-    # e.g. part_447__subpart_A__sec_1__para_b
+    # e.g. part_447__subpart_A__sec_1__para_b.
     identifier = models.CharField(max_length=1024)
     # e.g. para
     node_type = models.CharField(max_length=64)
     # e.g. b
-    type_emblem = models.CharField(max_length=16)
+    type_emblem = models.CharField(
+        max_length=16,
+        validators=[RegexValidator(
+            r'[A-Za-z0-9]+',
+            message="Only alphanumeric characters are allowed."
+        )]
```

Comment:
It's interesting to note that this validator, when used on `DocNode`, doesn't actually do anything useful, because we only ever bulk create `DocNode`s and never validate their fields.  It's only when we reuse this validator in our `DocCursorSerializer` that the logic is used.  So we could potentially _only_ include this in `DocCursorSerializer`, but I thought it'd be useful to have `DocNode` as its primary home to help developers understand the core model better.